src/reducers/ChatDux.js

```diff
@@ -30,8 +37,28 @@ const chat = createSlice({
       // eslint-disable-next-line no-param-reassign
       state.chatRooms = _orderBy(state.chatRooms, ['createdAt'], ['desc']);
     },
-    acceptOffer: () => {
-      // TODO: set offer message in chatConversation.conversation to ACCEPTED
+    acceptOffer: (state, { payload }) => {
+      const { chatRoomId, newChat, updatedAt, isDealClosed } = payload;
+      const messageIndex = state.chatConversation.conversation.findIndex(
+        c => c.id === newChat.id
+      );
+      const chatRoomIndex = state.chatRooms.findIndex(
+        c => c.chatRoomId === chatRoomId
+      );
+
+      // update chatConversation offer timestamp
+      // eslint-disable-next-line no-param-reassign
```

Comment:
Disable the rule from the whole block maybe, now it's just cluttering the code.

```
/* eslint-disable no-param-reassign */
const chat = createSlice({
....
}
/* eslint-enable no-param-reassign */
```

Remove all `// eslint-disable-next-line no-param-reassign` comments.